contracts/libraries/PoolSpecs.sol

```diff
@@ -60,6 +63,8 @@ library PoolSpecs {
         uint8 jitThresh_;
         uint8 knockoutBits_;
         uint8 oracleFlags_;
+        int24 bidTick_;
+        int24 askTick_;
```

Comment:
These should actually be target factors based on if base, quote implement decimals() (default 18 if they don't) versus discrete values so that the pool template supports assets with differing decimals